lib/_stream_wrap.js

```diff
@@ -110,18 +110,18 @@ StreamWrap.prototype.doShutdown = function doShutdown(req) {
 
 StreamWrap.prototype.doWrite = function doWrite(req, bufs) {
   const self = this;
-  const handle = self._handle;
+  const handle = this._handle;
 
   var pending = bufs.length;
 
   // Queue the request to be able to cancel it
-  const item = self._enqueue('write', req);
+  const item = this._enqueue('write', req);
 
-  self.stream.cork();
-  bufs.forEach(function(buf) {
-    self.stream.write(buf, done);
+  this.stream.cork();
+  bufs.forEach((buf) => {
+    this.stream.write(buf, done);
   });
```

Comment:
can you please convert this `forEach` in a for loop, it can probably save us a couple of ticks.